tests/interfaces/test_autograd_qnode.py

```diff
@@ -500,16 +499,13 @@ def test_differentiable_expand(self, interface, dev_name, diff_method, grad_on_e
             tol = TOL_FOR_SPSA
 
         class U3(qml.U3):
-            def expand(self):
+            def decomposition(self):
                 theta, phi, lam = self.data
                 wires = self.wires
-
-                with qml.queuing.AnnotatedQueue() as q_tape:
-                    qml.Rot(lam, theta, -lam, wires=wires)
-                    qml.PhaseShift(phi + lam, wires=wires)
-
-                tape = QuantumScript.from_queue(q_tape)
-                return tape
+                return [
+                    qml.Rot(lam, theta, -lam, wires=wires),
+                    qml.PhaseShift(phi + lam, wires=wires),
+                ]
```

Comment:
It appears this is coming back a lot, should we define and import that from some other module?

Reply:
So yes, we do need to test once for every interface, once for old default qubit and once for new default qubit, and once for new return types and once for old return types.

So that does add up into a lot of tests that look nearly identical.

But I'm fine with verbosity in tests, as they are often easier to debug when they fail.  And while I did have to make this change in a bunch of places, it was a fairly easy change to make.